activiti-core/activiti-bpmn-model/src/main/java/org/activiti/bpmn/model/BaseElement.java

```diff
@@ -1,144 +1,135 @@
 /*
- * Copyright 2010-2020 Alfresco Software, Ltd.
+ * Copyright 2019 Alfresco, Inc. and/or its affiliates.
```

Comment:
Please, revert this change in the header and update the header of `BaseElementTest` to match this one.